backend/remote/backend_plan_test.go

```diff
@@ -0,0 +1,181 @@
+package remote
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/backend"
+	"github.com/hashicorp/terraform/config/module"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/mitchellh/cli"
+)
+
+func testOperationPlan() *backend.Operation {
+	return &backend.Operation{
+		Type: backend.OperationTypePlan,
+	}
+}
+
+func TestRemote_planBasic(t *testing.T) {
+	b := testBackendDefault(t)
+
+	mod, modCleanup := module.TestTree(t, "./test-fixtures/plan")
```

Comment:
I'll have a look, but I would say that deserves a PR on its own then (to update that everywhere). As now it would stand out if I do it here, while no other backends tests currently do that... Something with consistency :wink: yet I totally get your point 👍

Reply:
Good point, consistency first 👍